admin/settings/class-admin-apple-settings-section-formatting.php

```diff
@@ -288,6 +288,28 @@ function __construct( $page ) {
 				'label' => __( 'Pull quote transformation', 'apple-news' ),
 				'type' => array( 'none', 'uppercase' ),
 			),
+			'monospaced_font' => array(
+				'label' => '',
```

Comment:
Yeah, that's the convention for the font dropdown on all fields that have fonts. It's a good point, though. I'll add it in.